oe_multilingual.post_update.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for OpenEuropa Multilingual module.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Invalidate service containers.
+ *
+ * Invalidate service containers for applying changes of
+ * oe_multilingual.local_translations_batcher service.
+ */
+function oe_media_post_update_invalidate_containers_cache(): void {
```

Comment:
Needs to comply with our [naming conventions](https://github.com/openeuropa/documentation/blob/master/docs/architecture/third-party/drupal/module-update-policy.md).

Reply:
Yes. fixed. sorry